### PR TITLE
Finalize State of Our Networks material

### PR DESCRIPTION
**Pull request by dcwalk**

### CODE REVIEW

File: state-of-our-networks.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The venue is an accessible space, for more details please see the <a href="http:
...
 | 9:00                | **REGISTRATION OPENS**                                                                                                                                                                                                       |
 | 9:15 &ndash; 9:45   | Introduction to mesh and community wireless <br /> _Intro led by [Dawn Walker](#dawn)_                                                                                                                                       |
-| 9:45 &ndash; 11:00  | [DIY Low Power Networks](https://github.com/tomeshnet/mini-conf-2017/blob/master/sessions/session-b.md) <br /> _Workshop led by [Udit Vera](#udit)_                                                                          |
+| 9:45 &ndash; 11:00  | [DIY Low Power Networks](https://github.com/tomeshnet/mini-conf-2017/blob/master/sessions/session-b.md) <br /> _Workshop led by [Udit Vira](#udit)_                                                                          |
```
Comment (benhylau) [inline state-of-our-networks.md:79]: @uditvira you're welcome

**Comment by benhylau** [comment]
Sorry I tried to fix things to reflect the doc move and made all the CI fail. Now I just piled all my changes into this branch.

File: state-of-our-networks.md (hunk 1 above)
Comment (dcwalk) [reply state-of-our-networks.md:79]: Ugh, I'm sorry I don't know why that typo always happens :(

**Comment by dcwalk** [comment]
No worries, I do the same thing @benhylau :(

**Pull request merged**